apps/dashboard/src/components/embedded-wallets/Configure/index.tsx

```diff
@@ -781,12 +797,17 @@ function NativeAppsFieldset(props: {
   );
 }
 
-function AdvancedConfigurationContainer(props: {
+function GatedCollapsibleContainer(props: {
   children: React.ReactNode;
-  show: boolean;
+  isExpanded: boolean;
   className?: string;
+  requiredPlan: Team["billingPlan"];
+  currentPlan: Team["billingPlan"];
 }) {
-  if (!props.show) {
+  const upgradeRequired =
+    planToTierRecord[props.currentPlan] < planToTierRecord[props.requiredPlan];
```

Comment:
this comparison won't work because `growth_legacy` is treated as "lower" than `growth` - but in this case we need it to be treated as higher - I think it's probably safe to switch it in the "tier list"

Reply:
I just added a new record where growth_legacy is even higher than accelerate